Search-algorithms/src/base/Counting_sort.java

```diff
@@ -1,10 +1,14 @@
 package base;
 
-public class Counting_sort extends SorterAlgorithm {
+import utility.*;
 
-	@Override
-	public Integer[] sorting(Integer[] arrayToBeOrder,Integer[] OrderArray, Integer maxValue ) {
+public class Counting_sort extends SorterAlgorithm {
```

Comment:
il nome della classe deve essere in cammel case in modo da facilitare eclipse nel lavoro di context assist